### ISSUE TITLE
Errors with windows Dropbox content

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/osm2pgsql#719]
cc @alex85k 

```
curl -O https://dl.dropboxusercontent.com/u/63393258/libs_osm2pgsql_%Configuration%.7z
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3309    0  3309    0     0  18662      0 --:--:-- --:--:-- --:--:-- 21487
7z x libs_osm2pgsql_%Configuration%.7z | find ":"
ERROR: libs_osm2pgsql_Release.7z
libs_osm2pgsql_Release.7z
Open ERROR: Can not open the file as [7z] archive
```

https://dl.dropboxusercontent.com/u/63393258/libs_osm2pgsql_Release.7z returns a 404 error.

**Comment by lonvia** [comment]
To recreate the library package we would need:

 - the full dependencies of libosmium (see https://github.com/mapbox/windows-builds)
 - Postgres client libary (libpq)
 - lua library
 - Boost: base libraries, system filesystem date_time chrono

**Comment by joto** [comment]
Something in https://github.com/alex85k/winbuilds might be helpful here.

**Comment by BergWerkGIS** [comment]
Did a quick test run with [mapbox/windows-builds](https://github.com/mapbox/windows-builds) for x64:

```bat
git clone https://github.com/mapbox/windows-builds windows-builds-64
cd windows-builds-64
settings "FASTBUILD=0"
:: libosmium_deps include boost: system, filesystem, date_time, chrono
scripts\build_libosmium_deps.bat
:: creates a 7z in packages\libosmium-deps
scripts\package_libosmium_deps.bat
:: libs will be in packages\lua\build\RelWithDebInfo
scripts\build_lua.bat
:: libs will be in packages\postgresql\src\interfaces\libpq\Release
scripts\build_libpq.bat
```


To do a x86 build:

```bat
git clone https://github.com/mapbox/windows-builds windows-builds-32
cd windows-builds-32
settings "FASTBUILD=0" "TARGET_ARCH=32"
:: other commands are the same
```


The resulting files of the above commands (x64) can be downloaded here:
https://www.dropbox.com/sh/2sjq5o93wtmb508/AADT5PWvp5omC-A_2hHzRLD_a?dl=0

They will require some more work to massage the files/folders into the structure required by the osm2pgsql scripts.

**Comment by lonvia** [comment]
Thanks. The lua library was missing the headers but I could borrow those from the OSRM build package. That got me pretty far, see #742. Now it seems to be missing data for the proj library. There is a `proj/share/epsg` in the libosmium dependency package and I've put it into `share/epsg` but that doesn't make a difference. Do you happen to know where windows would look for this file?

**Comment by BergWerkGIS** [comment]
@lonvia you could try setting the `PROJ_LIB` env var:

> The PROJ_LIB environment variable can be used to specify the directory where the PROJ.4 data files (including the "epsg" file) are located, if they are not in the default directory where PROJ.4 expects them.

**Comment by alex85k** [comment]
I am available now if needed, but using https://github.com/mapbox/windows-builds would be better, because it is used in real development and more tested.

**Comment by lonvia** [comment]
Fixed in #742.
